### DISCUSSION TITLE
Highlights and annotations?

### DISCUSSION THREAD

**Discussion by thefathermucker** [edrlab/thorium-reader#1995] [Ideas]
Hi, I'm a new convert to Thorium. Thank you EDR Lab for creating such a wonderful ebook reader.

Are there plans for highlights and annotations in an upcoming version? This is the only major feature missing IMHO and would make the app perfect.

Thanks!

**Comment by panaC** [comment]
Hello, thank you for your interest.

yes there a plan. It will be integrated in next release thorium-2.4. You can find our work here #1216 and issue #1537

**Reply by thefathermucker** [reply]
Wonderful. Thank you so much!
